app/src/main/java/com/example/harvestfresh/fragments/EventsFragment.java

```diff
@@ -125,20 +126,18 @@ private void goPlace(LatLng searchLatLng, String placeName) {
     private void placeMarkers(ParseGeoPoint userLocation) {
```

Comment:
onViewCreated is doing several doing things, can we atomize them a little better?